packages/playground/src/App.vue

```diff
@@ -215,26 +205,23 @@ const theme = useTheme();
 const navbarConfig = ref();
 
 const hasGrid = computed(() => !!gridStore.grid);
+
 watch(
   () => $route.meta,
   meta => {
     (document.title = "Threefold Dashboard" + (meta && "title" in meta ? ` | ${meta.title}` : ``)),
       (navbarConfig.value = meta.navbarConfig);
   },
 );
+function navigateToHome() {
+  return $router.push("/");
+}
```

Comment:
Please revemoe `navigateToHome` if it's not used anywhere?

Reply:
it is used on line 119